locales/en/app.ftl

```diff
@@ -114,18 +114,28 @@ form-signup-error = Must be a valid email
 found-breaches-headline = This could be a problem
 no-breaches-headline = So far, so good
 
+-no-breaches = 
```

Comment:
This shouldn't be a term. You can [reference standard messages](https://projectfluent.org/fluent/guide/references.html) in other messages.